torch/csrc/jit/passes/remove_mutation.cpp

```diff
@@ -263,6 +263,11 @@ struct MutationRemover {
   std::shared_ptr<Graph> graph_;
 };
 
+bool isInplaceOpVariant(const std::shared_ptr<Graph>& graph, Node* node) {
+  MutationRemover mr(graph);
```

Comment:
This is going to construct an alias DB on each invocation, which will be super slow. As such I think it's better not to expose `isInplaceOpVariant` as a public API except as a method on the class, and in whatever pass relies on this have it instantiate a single MutationRemover and query that one instance `inplaceOpVariant`. 

In the future it would be nice to decouple the alias schema querying from instantiating the alias db, but that's a good amount of work and probably not needed for this PR

Reply:
Thanks @eellison 
So for now we can use the MutationRemover aliasDb. I just made the change.